### PR TITLE
Rblandingpage

### PR DESCRIPTION
**Pull request by rryanburton**
linked username to user profile in navbar
added score spot in navbar

added landing page

moved recent questions to its own page

### CODE REVIEW

File: question_box/questionapp/templates/base.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
              <button type="submit" class="btn btn-warning">Sign out</button>
              <a class="btn btn-success" href="/question/new" role="button">New Question &raquo;</a>
            </form>
-           <span class="navbar-right navbar-brand ">Welcome {{ request.user }}</span>
+           <span class="navbar-right navbar-brand ">Welcome <a href="/profile/{{ request.user.id }}">{{ request.user }}</a> <span class="badge">42 Points</span></span>
```
Comment (gmhurley) [inline question_box/questionapp/templates/base.html]: So, right now the Welcome highlights when hovered. I'd say we either need to get rid of that, or just make the Welcome part of the link. 

Also, and we remove the coloring on the link? Turned the name blue. Think we should try to leave the styling like the template as much as possible.

**Comment by nckathleen** [comment]
Upon logout user should be taken to the landing page.  Seems like it is currently staying where ever the user logged out from.

**Comment by rryanburton** [comment]
i agree with the sign-out button action, but it's not in the scope of this commit. we can add that to the issues list for a future feature

**Comment by nckathleen** [comment]
Ok, then I'll merge it.

**Pull request merged**